scripts/forecast_csv.py

```diff
@@ -6,7 +6,41 @@
 import unittest
```

Comment:
what do you think about moving this to the main library? and adding a test for this?
 Im sometimes a bit worried if we depend to much on `scripts`

Reply:
Yeah I can look into that. Shall I do that in a future PR though? I feel like it's beyond the scope of this change.

Reply:
Thanks for adding the test, yea that could be a future PR

Reply:
I ended up moving forecast_csv along with hf_upload to a new utils module